frontend/src/stories/components/Chat/ChatToggle.stories.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import ChatToggle from "main/components/Chat/ChatToggle";
+
+export default {
+    title: 'components/Chat/ChatToggle',
+    component: ChatToggle
+};
+
+
+const Template = (args) => {
+    return (
+        <ChatToggle {...args} />
+    )
+};
+
+const toggleChatWindow = () => {}
```

Comment:
This might help when testing/demostrating the component:

```suggestion
const toggleChatWindow = () => {
   window.alert("chat toggle clicked");
}
```